### DISCUSSION TITLE
Can the GPU speed up the display of *.bim files ?

### DISCUSSION THREAD

**Discussion by namsancow** [iTwin/itwinjs-core#1474] [Q&A] [answered]
Tiled *.bim files made of 3D realty meshes are displayed quickly.
However, the larger the size of *.bim file, the slower the display speed. 
In particular, when *.bim file are published from texture models in *.obj or *.fbx format, their display slows significantly.
From my experience so far, I know that RAM must be at least 16GB to avoid "Ooops" errors during the display process. 
However, I also know that increasing the RAM size does not increase the display speed.
Can the GPU speed up the display of *.bim files ?

**Comment by pmconne** [comment, marked as answer]
Please define "tiled *.bim files made of 3D reality meshes". How are you creating these? Are you putting the 3d tiles into the .bim, or only their URLs?
What application are you using to display the iModels? FBX files are notorious for using extremely large textures; that might explain high memory usage.
For further assistance please send some examples to pc4868696@gmail.com

**Reply by namsancow** [reply]
I sent email to pc4868696@gmail.com for your questions.
Many thanks to your kind response as always.

**Reply by pmconne** [reply]
Via email we determined the problem is caused by downsampling of 4k textures produced from .fbx file. We have a plan to address this in 2.17.0.

**Reply by pmconne** [reply]
Downsampling solution was implemented.
